### PR TITLE
[TE] Send out notification upon exceptions

### PR DESCRIPTION
**Pull request by cyy0714**
Force onboarding job sending out emails upon exceptions

### CODE REVIEW

File: thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/onboard/tasks/FunctionCreationOnboardingTask.java [hunk 1]
```diff
@@ -271,6 +271,7 @@ public void run() {
       alertConfig.setApplication(configuration.getString(ALERT_APPLICATION));
       alertConfig.setRecipients(alertRecipients);
       alertConfig.setCronExpression(configuration.getString(ALERT_CRON, DEFAULT_ALERT_CRON));
+      alertConfig.setActive(DEFAULT_IS_ACTIVE);
```
Comment (apucher) [inline thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/onboard/tasks/FunctionCreationOnboardingTask.java]: Are you sure about this? This means any alert, even if backtesting fails, will start eating up cycles

Comment (cyy0714) [reply thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/onboard/tasks/FunctionCreationOnboardingTask.java]: When do you think it is a proper place to activate the new alert?

Comment (apucher) [reply thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/onboard/tasks/FunctionCreationOnboardingTask.java]: after it successfully completes backtesting, at least

Comment (cyy0714) [reply thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/onboard/tasks/FunctionCreationOnboardingTask.java]: Thanks for the comment. It's fixed.

**Pull request merged**